doc/update065.md

```diff
@@ -21,3 +21,25 @@ tail -f ~/.komodo/VRSC/debug.log
 ```
 
 Make sure chain is running smoothly and notarisations are progressing.
+
+- Remove SFUSD.
+```bash
+# Return remaining SFUSD funds
+cd ~/sfusd-core/src
+./smartusd-cli sendtoaddress SVpijx2FxX67ny8LAtC4ogrvpSWzwkda5x $(./smartusd-cli getbalance) "" "" true
```

Comment:
i guess returning of the funds would work better if iguana would be restarted first, else some SFUSD UTXOs may be locked by iguana

Reply:
done